tests/setup-tests.js

```diff
@@ -0,0 +1,6 @@
+import 'regenerator-runtime'
+import { configure } from 'enzyme'
```

Comment:
FYI I think enzyme is no longer maintained.

Reply:
TLDR: We need enzyme to check the state of a class based component.

Good point, but since we are using a class based component state, I needed enzyme to check the state since react-testing-library shoots more for functional components and hooks, those being easier to mock and test.
Maybe in a future refactor when we move out of class based components we restructure the test to use react-testing-library.